translations/de/info/task_description.html

```diff
@@ -0,0 +1,59 @@
+<div class="story">
+    <div style="text-align: center">
+        <img src="{{MEDIA}}robots.png" style="max-height: 350px">
+    </div>
+    <p>
+        Mit 50 verschiedenen Steckertypen findet man keine passenden Geräte oder diese würden sehr teuer sein.
+        Sobald aber ein elektrischer Anschluss standardisiert wird, können viele Firmen Geräte designen,
+        Wettbewerb entsteht, mit wachsender Auswahl und besseren Preisen für Konsumenten.
+        <br>
+        -- Bill Gates
+
+    </p>
+
+</div>
+
+<p>
+    Du erhältst eine nicht-leere Liste von Ganzzahlen (X).
+    Für diese Aufgabe, solltest Du eine Liste bestehend nur aus den nicht-einmaligen Elementen daraus zurückgeben.
+    Dazu musst Du alle einmaligen Elemente (Elemente welche in einer gegebenen Liste nur genau einmal vorkommen) entfernen.
+    Beim Lösen dieser Aufgabe darfst Du die Reihenfolge der Liste nicht verändern.
+    Beispiel: [1, 2, 3, 1, 3] 1 und 3 sind nicht-einmalige Elemente und das Resultat lautet [1, 3, 1, 3].
+</p>
+
+<p style="text-align: center;">
+    <img title="non-unique-elements"
+         src="{{MEDIA}}non-unique-elements.png"
+         alt="non-unique-elements"
+         style="max-height: 160px"/>
+</p>
+
+<p><strong>Eingabe: </strong>Eine Liste von Ganzzahlen.</p>
+
+<p><strong>Ausgabe: </strong>Die Liste von Ganzzahlen.</p>
+<div class="for_info_only">
+    <p><strong>Beispiel:</strong></p>
+<pre class="brush: python">{% if interpreter.slug == "js-node" %}
+nonUniqueElements([1, 2, 3, 1, 3]) == [1, 3, 1, 3]
+nonUniqueElements([1, 2, 3, 4, 5]) == []
+nonUniqueElements([5, 5, 5, 5, 5]) == [5, 5, 5, 5, 5]
+nonUniqueElements([10, 9, 10, 10, 9, 8]) == [10, 9, 10, 10, 9]
+{% else %}
+checkio([1, 2, 3, 1, 3]) == [1, 3, 1, 3]
+checkio([1, 2, 3, 4, 5]) == []
+checkio([5, 5, 5, 5, 5]) == [5, 5, 5, 5, 5]
+checkio([10, 9, 10, 10, 9, 8]) == [10, 9, 10, 10, 9]
+{% endif %}</pre>
+</div>
+<br>
+<p>
+    <strong>Wozu das gut ist:</strong>
+    Diese Mission hilft Dir zu verstehen wie man Arrays (DatenFelder) manipuliert,
```

Comment:
"Datenfeld" is not the german word "array", basically there is no need to translate array.